pkg/cloud/services/awsnode/service.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package awsnode
 
 import (
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
+	corev1 "k8s.io/api/core/v1"
 	infrav1 "sigs.k8s.io/cluster-api-provider-aws/api/v1beta1"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
So for this linter error, make lint-fix was supposed to fix the import order automatically but looks like gci's latest upgrade broke it - https://github.com/golangci/golangci-lint/issues/2604

As defined https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/.golangci.yml, we group imports in three parts. 

```suggestion
         corev1 "k8s.io/api/core/v1"
	"sigs.k8s.io/controller-runtime/pkg/client"
	
	infrav1 "sigs.k8s.io/cluster-api-provider-aws/api/v1beta1"
	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud"
```

Reply:
Looks like that worked 🎉, thanks @pydctw